### PR TITLE
payment crud done

### PR DESCRIPTION
**Pull request by Ronakambaliya**
payment crud done

### CODE REVIEW

File: api/models/payment.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+// models/Payment.js
+
+const pool = require('../db/db');
+
+class Payment {
+  static async create(paymentData) {
+    const { userId, money, paymentAt } = paymentData;
+    // Insert new payment
+    const insertQuery = 'INSERT INTO payments (id, money, payment_at) VALUES ($1, $2, $3) RETURNING *';
```
Comment (Uttamjethva77) [inline api/models/payment.js:9]: user_id colum where it?

File: api/models/payment.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+// models/Payment.js
+
+const pool = require('../db/db');
+
+class Payment {
+  static async create(paymentData) {
+    const { userId, money, paymentAt } = paymentData;
```
Comment (Uttamjethva77) [inline api/models/payment.js:7]: make user_id

File: api/models/payment.js (hunk 1 above)
Comment (Ronakambaliya) [reply api/models/payment.js]: changes done

File: api/models/payment.js (hunk 2 above)
Comment (Ronakambaliya) [reply api/models/payment.js]: changes done

**Review by Uttamjethva77 — COMMENTED** [review]
solve conflict

**Review by Uttamjethva77 — APPROVED** [review]

**Pull request merged**